### PR TITLE
Bug Squash! Jasmyn, Grace and I fixed the issue of the site not displ…

### PR DESCRIPTION
**Pull request by RosaTheDev**
…aying information on reload

____ Wrote Tests  x Implemented ____ Reviewed

## Necessary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [] New feature
    [x] Bug Fix
    [] Refactor

## Description of Change:

    description: Jasmyn Grace and I fixed the issue where the information would not load upon site refresh. Changed useContext with useParams
    
## Requested feedback: 
    
    I'd like feedback on... 

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [x] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -8,13 +8,11 @@ import SingleSighting from './Components/SingleSighting'
 import NavBar from './Components/NavBar';
 import ErrorModal from './Components/ErrorModal'
 import { NavigationProvider } from './Context/NavigationContext';
-import { CryptidProvider } from './Context/CryptidContext';
```
Comment (GraceGardner) [inline src/App.js:11]: Thanks for cleaning up this file

**Review by jasmyn2244 — APPROVED** [review]
Love what we are learning about context vs useParams and which situations to use which one.

File: src/Components/SightingCard.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import '../Styles/Card.scss';
 import React from 'react';
 import { Link } from 'react-router-dom'
-import bigfoot from '../assets/bigfoot.jpg'
```
Comment (GraceGardner) [inline src/Components/SightingCard.js:4]: good catch

File: src/Components/SingleCryptid.js [hunk 3]
```diff
@@ -3,8 +3,7 @@ import '../Styles/SingleCryptid.scss'
 import { NavigationContext } from '../Context/NavigationContext'
 import FootPrints from './FootPrints'
 import { gql, useQuery } from '@apollo/client'
-import { Link } from 'react-router-dom'
-import { CryptidContext } from '../Context/CryptidContext'
+import { Link, useParams } from 'react-router-dom'
```
Comment (GraceGardner) [inline src/Components/SingleCryptid.js:6]: awesome change! The reload is working now!!!

File: src/Components/SingleSighting.js [hunk 4]
```diff
@@ -51,7 +51,7 @@ const SingleSighting = () => {
 
   const trailLinks = data.sightingById.trailLinks.map((link, index) => {
     return (
-      <a className='trail-links' href={link} key={index} target='_blank'>Trail {index + 1}</a>
+      <a className='trail-links' href={link} key={index} target='_blank' rel="noreferrer">Trail {index + 1}</a>
```
Comment (GraceGardner) [inline src/Components/SingleSighting.js:54]: Good catch Richard!

**Review by GraceGardner — APPROVED** [review]
Good work! That was a quick fix! Crushed it!

**Pull request merged**